php/libraries/LorisForm.class.inc

```diff
@@ -1352,11 +1352,11 @@ class LorisForm
      *
      * @param callable $func The callback to use to process this form.
      *
-     * @return none
+     * @return mixed Preferably `bool`...
```

Comment:
But I can have the check in this method instead, if this is the behaviour we want to enforce.